jmh-core/src/main/java/org/openjdk/jmh/generators/core/StateObjectHandler.java

```diff
@@ -966,7 +966,9 @@ public void writeStateOverrides(BenchmarkGeneratorSession sess, GeneratorDestina
                 pw.println("import " + so.userType + ";");
 
                 pw.println("public class " + so.type + "_B1 extends " + so.userType + " {");
-                Paddings.padding(pw);
+                // The prefix for padding should be different from the one used in _B3 to avoid
+                // "hiding field of the same name in superclass" warning
```

Comment:
You don't need these comments, the issue history has enough breadcrumbs about this.